front/src/modules/trainschedule/components/ManageTrainSchedule/PowerRestrictionsSelector.tsx

```diff
@@ -89,6 +91,9 @@ const PowerRestrictionsSelector = ({
     [rollingStockModes]
   );
 
+  const { updatePowerRestrictionRanges } =
+    useOsrdActions() as operationalStudiesConfSliceActionsType;
+
```

Comment:
Could you group it with the other custom hook line 59 :)

Reply:
Done